awx/main/utils/common.py

```diff
@@ -92,14 +92,20 @@ def camelcase_to_underscore(s):
     return s.lower().strip('_')
 
 
-class RequireDebugTrueOrTest(logging.Filter):
+class FilterByDebugOrLogLevel(logging.Filter):
     '''
     Logging filter to output when in DEBUG mode or running tests.
     '''
 
     def filter(self, record):
         from django.conf import settings
-        return settings.DEBUG or settings.IS_TESTING()
+        if settings.DEBUG or settings.IS_TESTING():
+            return True
+        try:
+            allowed_level = settings.LOG_AGGREGATOR_LEVEL
+        except Exception:
```

Comment:
Usually I don't like catching generic `Exception` because it can hide some real issues. Is there any particular exceptions you are looking for or do you think this would not harm in here?

Reply:
A _lot_ can technically go wrong here, but it's mostly related to a number of database and/or memcached connectivity issues.  In the _unlikely_ event we fall into this except block, I just want to have awx degrade nicely here.  I'm afraid that itemizing the list of possible exceptions is going to get _really_ tedious.